src/style.css

```diff
@@ -1,3 +1,81 @@
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+p {
+  margin-top: 0;
+  margin-bottom: 0;
+}
+
+ul,
+ol {
+  padding-left: 0;
+  margin-top: 0;
+  margin-bottom: 0;
+}
```

Comment:
Avoid using tag names for styling. Use classes to style these elements, especially since you have already started using classes for other elements like `.moyo_menu`.